apps/frontend/src/components/containers/Auth/Shop/Carousel/CarouselItem/CarouselItem.view.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import classes from './CarouselItem.module.scss';
+
+type TProps = {
+	readonly imageSrc: string;
+	readonly alt: string;
+	readonly title: string;
+	readonly text: string;
+};
+const CarouselItemView = (props: TProps) => {
+	return (
+		<div className={classes['container']}>
+			<img src={props.imageSrc} alt={props.alt} className={classes['container__image']} />
+			<div className={classes['innerContainer']}>
```

Comment:
In another place you called this type of class - `carouselContainer`.

You should follow a pattern when it comes to class names, I would go for `{informativeName}Container` instead of `innerContainer` since that's less informative.